src/otx/v2/adapters/datumaro/adapter/action_dataset_adapter.py

```diff
@@ -156,9 +153,8 @@ class ActionDetectionDatasetAdapter(ActionBaseDatasetAdapter):
 
     def get_otx_dataset(self) -> DatasetEntity:
         """Convert DatumaroDataset to DatasetEntity for Acion Detection."""
-        label_information = self._prepare_label_information(self.dataset)
-        self.label_entities = label_information["label_entities"]
-
+        if not hasattr(self, "label_entities"):
+            super().get_label_schema()
```

Comment:
We need to remove this

Reply:
Thanks ! https://github.com/openvinotoolkit/training_extensions/pull/2631/commits/27bd09929221077fede025123dda1cee8d28f69f